src/lib/local-api/devices.ts

```diff
@@ -0,0 +1,48 @@
+import { Logs as LogsModel } from '../xively/models/logs';
```

Comment:
if we are going to bring in these files we have to modify them to work for this context.  Looks like as it stands these will send requests to relative URL's and we need to make them absolute using the config setting for api url